docs/src/globals.css

```diff
@@ -79,6 +79,7 @@
  */
 .references-layout {
   height: max-content;
+  width: 100% !important;
```

Comment:
Linked doc did not mention this. Is it required, or was this accidentally left in from testing?

Reply:
> Linked doc did not mention this. Is it required, or was this accidentally left in from testing?

@markflorkowski good catch. So what I didn't realize that the "full" layout strips away the container, breadcrumbs, etc... which is why the width was necessary to override the 100dvw.

I've made a change to remove just the `toc` via the configuration, based on their design decisions they mark the element as hidden so it would still retain the space.

Before: 
![image](https://github.com/pingdotgg/uploadthing/assets/2293095/21a097b6-18bf-4304-9a1d-25d51aefadf0)

After:
![image](https://github.com/pingdotgg/uploadthing/assets/2293095/5154390a-a40d-490e-9a74-53592a38a712)